module/src/components/checkboxInput/checkboxInput.component.tsx

```diff
@@ -57,7 +58,7 @@ export const CheckboxInput = React.forwardRef<HTMLInputElement, ICheckboxInputPr
       checked,
       onChange,
       checkedIcon,
-      content,
+      label,
```

Comment:
This is a breaking change. Is it worth a full version increase for just this?

Reply:
It was something that was broken in a recent release by accident, so this is just changing it back. I'm personally fine with it and it's come out of someone raising it specifically.

Reply:
But also happy to switch it back

Reply:
yeah, I only worry that if people have fixed it they'll have to unfix it again... It's hard to know what the better option is tbh

Reply:
Talking to Ben, apparently everyone has already fixed it in their code so probably worth rolling it back to content